app/services/complexity_event_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class ComplexityEventService
+  class << self
+    def send_complexity complexity
+      sns_topic.publish(
+        message: {
+          offenderNo: complexity.offender_no,
+          level: complexity.level,
+        }.to_json,
+        message_attributes: {
+          eventType:  {
+            string_value: "complexity-of-need.level.changed",
+            data_type: "String",
+          },
+          version: {
+            string_value: 1.to_s,
+            data_type: "Number",
+          },
+          occurredAt: {
+            string_value: complexity.created_at.to_s,
+            data_type: "String",
+          },
+          detailURL: {
+            string_value: "#{Rails.configuration.complexity_of_need_host}/complexity-of-need/offender-no/#{complexity.offender_no}",
+            data_type: "String",
+          },
+        },
+        )
+    end
+
+    def spike_subscribe
+      sns_topic.subscribe attributes: { FilterPolicy: { eventType: ["complexity-of-need.level.changed"] }.to_json },
+                          protocol: "email",
+                          endpoint: "stephen.dicks@digital.justice.gov.uk"
+    end
```

Comment:
Just highlighting this as we'll want to remove it before incorporating this into `main`.